### DISCUSSION TITLE
Use with Standard Oscilloscope Probe (x10)

### DISCUSSION THREAD

**Discussion by Simba185** [fhdm-dev/scoppy#53] [Scoppy Front end designs]
So I have built a few of the analog front end designs, thanks for supplying those.

Is there a way to use them with a standard scope probe?

I have tested this ( works ok with x1 mode but x10 mode is incorrect divider and capacitance match)

Any ideas on what can be changed in the design? If the x10 probe is direct to pico max voltage is 33v.  IMO  If you use the op amp input then the x10  probe voltages more along the lines of 120v (185v) would be safe. 
Scope front end should be 1M ohm and 117pf to match most standard x10 probes.

Thanks.

**Comment by gilesp1729** [comment]
With the circuits using the resistor divider to Vref to level shift the input, wouldn't the Vref have to change with a x10 probe?

**Comment by fhdm-dev** [comment]
I'm not sure if it's possible with those AFEs without changing the value of Vref when the probe changes from 1x to 10x and back (as @gilesp1729 mentioned). If you did want to go down that path the first thing to do is to change the total resistance of the input divider to 1M.
For example with Design #1, changing the resistor values to 725k and 275k would do it. Vref is then 2.28V. When changing to 10x, Vref would then need to be 1.70V - assuming my calculations are correct :)
